### PR TITLE
Branch 1.11

### PR DESCRIPTION
**Pull request by pjgg**

### CODE REVIEW

File: 303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public void checkBorderConditionBetweenIdleAndGetConnection(){
                                     }
                                     return result;
                                 }).toMulti();
-                    }).collect().in(ArrayList::new, List::add).subscribe().with(re -> {
+                    }).collectItems().in(ArrayList::new, List::add).subscribe().with(re -> {
```
Comment (rsvoboda) [inline 303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java:164]: What triggers `.collect() => collectItems()` ?

File: pom.xml [hunk 2]
```diff
@@ -25,8 +25,8 @@
       <module>014-quarkus-panache-with-transactions-xa</module>
       <module>015-quarkus-micrometer</module>
       <module>017-quartz-cluster</module>
-      <module>018-quarkus-runtime-properties</module>
-      <module>019-quarkus-consul</module>
+     <!-- <module>018-quarkus-runtime-properties</module>
+      <module>019-quarkus-consul</module> -->
```
Comment (rsvoboda) [inline pom.xml:29]: +1 ... also noted in https://github.com/quarkus-qe/beefy-scenarios/pull/97#discussion_r579206302

File: pom.xml [hunk 3]
```diff
@@ -52,8 +52,8 @@
 
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-    <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
+    <quarkus.platform.version>1.11.3.Final</quarkus.platform.version>
+    <quarkus-plugin.version>1.11.3.Final</quarkus-plugin.version>
```
Comment (rsvoboda) [inline pom.xml:56]: +1

**Review by rsvoboda — CHANGES_REQUESTED** [review]
https://github.com/quarkus-qe/beefy-scenarios/blob/master/.github/workflows/pr.yaml#L11 ... 1.11 should be against 20.3

https://github.com/quarkus-qe/beefy-scenarios/blob/master/.github/workflows/pr.yaml#L24 ... no need to build master, we will run in 1.11 branch against 1.11.3 atm

Same adjustments to 1.11 equivalent of https://github.com/quarkus-qe/beefy-scenarios/blob/master/.github/workflows/daily.yaml


There are also 
 - https://github.com/quarkus-qe/beefy-scenarios/pull/99
 - https://github.com/quarkus-qe/beefy-scenarios/pull/98

which should make it into 1.11 branch (at least the spring data rest)

File: 303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java (hunk 1 above)
Comment (pjgg) [reply 303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java:164]: It's not supported on 1.11.3. The method `collectItems` has been replaced by `collect()` on version 1.12

Comment (rsvoboda) [reply 303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java:164]: This should be highlighted at least in https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.12
Changes of Uni / Multi related API can be trouble in the future, it should really stabilize.
Please talk to Clement about `collectItems() => collect()`  shift in Quarkus 1.12 and suggest to enhance the migration guide. Another option is GH issue about it, but I would start with ping on zulip

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**